lib/matplotlib/axes/_axes.py

```diff
@@ -6986,12 +6956,6 @@ def psd(self, x, NFFT=None, Fs=None, Fc=None, detrend=None,
         r"""
         Plot the power spectral density.
 
-        Call signature::
-
-          psd(x, NFFT=256, Fs=2, Fc=0, detrend=mlab.detrend_none,
```

Comment:
The default `detrend` here says it's 'none', but the new docstring is a bit vague about it, and seems like it should be 'default' (which is 'mean'). I think the actual default _is_ 'none' though.

Reply:
Thanks for catching that.
I edited the PSD signature in mlab.py to not list "default" and "constant" anymore (and explicitly indicate that "none" is the default), with the plan to deprecate "default" and "constant" in a separate PR (having a default that's not "default" is just... confusing).